offchain-modules/packages/app-rpc-server/src/handler/burn/eth.ts

```diff
@@ -0,0 +1,37 @@
+import { ForceBridgeCore } from '@force-bridge/x/dist/core';
+import { stringToUint8Array } from '@force-bridge/x/dist/utils';
+import { logger } from '@force-bridge/x/dist/utils/logger';
+import { abi } from '@force-bridge/x/dist/xchain/eth/abi/AssetManager.json';
+import ethers from 'ethers';
+import { GenerateBridgeNervosToXchainBurnTxPayload, GenerateTransactionResponse } from '../../types/apiv1';
+import { NetworkBase } from '../../types/network';
+import Burn from './burn';
+
+class Eth extends Burn {
+  async handle<T extends Required<NetworkBase>>(
+    payload: GenerateBridgeNervosToXchainBurnTxPayload,
+  ): Promise<GenerateTransactionResponse<T>> {
+    logger.info(`generateBridgeOutEtherumTransaction, payload: ${JSON.stringify(payload)}`);
+
+    this.checkCKBAddress(payload.recipient);
```

Comment:
Miss `checkBurn`, ref: https://github.com/nervosnetwork/force-bridge/blob/85725f19cae4f5c36eacf8e351e85568ac56c6fa/offchain-modules/packages/app-rpc-server/src/handler.ts#L88

* check asset in tokenToAssetIdMap(AssetManager.sol)
* check amount >= minimalAmount
* check sudtSize <= ForceBridgeCore.config.ckb.sudtSize
* besides check recipientAddressFormat, better check recipientAddressLength <= dbRecipientFieldLength